src/pages/Vault/VaultStatusStatPanel/index.tsx

```diff
@@ -0,0 +1,179 @@
+
+import { useQuery } from 'react-query';
+import {
+  useErrorHandler,
+  withErrorBoundary
+} from 'react-error-boundary';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import Big from 'big.js';
+import { AccountId } from '@polkadot/types/interfaces';
+import { VaultExt } from '@interlay/interbtc-api';
+import { BitcoinUnit } from '@interlay/monetary-js';
+
+import StatPanel from '../StatPanel';
+import ErrorFallback from 'components/ErrorFallback';
+import { COLLATERAL_TOKEN } from 'config/relay-chains';
+import { getVaultStatusLabel } from 'utils/helpers/vaults';
+import { COLLATERAL_TOKEN_ID_LITERAL } from 'utils/constants/currency';
+import genericFetcher, { GENERIC_FETCHER } from 'services/fetchers/generic-fetcher';
+import { BTCToCollateralTokenRate } from 'types/currency.d';
+import { StoreType } from 'common/types/util.types';
+
+interface Props {
+  // TODO: should remove `undefined` later on when the loading is properly handled
+  vaultAccountId: AccountId | undefined;
+}
+
+const VaultStatusStatPanel = ({
+  vaultAccountId
+}: Props): JSX.Element => {
+  const { t } = useTranslation();
+  const { bridgeLoaded } = useSelector((state: StoreType) => state.general);
+
+  const {
```

Comment:
This is a comment as much for me as it is for you :) I've added a ticket to consider how we handle multiple api calls in a single component. It's something we have to do a lot across the site, and I think we can improve it. It's not related to this component specifically and it's not a high priority, I'm just flagging that's it been added to the backlog.